twitterpuller/src/main/java/de/brandwatch/minianalytics/twitterpuller/postgres/cache/QueryCache.java

```diff
@@ -0,0 +1,21 @@
+package de.brandwatch.minianalytics.twitterpuller.postgres.cache;
+
+import de.brandwatch.minianalytics.twitterpuller.postgres.model.Query;
+import de.brandwatch.minianalytics.twitterpuller.postgres.repository.QueryRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class QueryCache {
+
+    @Autowired
+    QueryRepository queryRepository;
```

Comment:
can be `private` or `private final` if you use constructor injection.

Reply:
Done